### PR TITLE
Feature/kiss the cook

### PR DESCRIPTION
**Pull request by JaXt0r**
# To test
1. Start game
2. Check if NPCs have different heads and body color
3. Check if Snaf is there (Hint: His head is slightly misplaced, but I will fix this later)

# Checklist
(Please judge for yourself which entry is valuable for your PR)

## Scene
* [x] FeatureFlags reverted to right value

# Testing
* [x] Merged _main_ into this branch and tested with the latest features
* [x] Tested with PCVR
* [ ] Tested with Pico / Quest

### CODE REVIEW

File: Assets/GothicVR/Scripts/Creator/Meshes/NpcMeshCreator.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            if (name.ToUpper().Contains("MOUTH") || name.ToUpper().Contains("TEETH"))
+                return base.GetTexture(name);
+            
+            if (!name.ToUpper().EndsWith("V0_C0.TGA"))
+            {
+                Debug.LogError($"The format of body texture isn't right for ${name}");
+                return base.GetTexture(name);
+            }
+
+            var formattedTextureName = Regex.Replace(name, "(?<=.*?)V0_C0", $"V{tmpBodyTexNr}_C{tmpBodyTexColor}");
```
Comment (JucanAndreiDaniel) [inline Assets/GothicVR/Scripts/Creator/Meshes/NpcMeshCreator.cs:63]: Can you add a comment on what the regex does? the first part might be confusing for people not very knowledgeable about regex

**Review by JucanAndreiDaniel — APPROVED** [review]
LGTM, just regex part needs explanation

File: Assets/GothicVR/Scripts/Creator/Meshes/NpcMeshCreator.cs (hunk 1 above)
Comment (JaXt0r) [reply Assets/GothicVR/Scripts/Creator/Meshes/NpcMeshCreator.cs:65]: Done. Thanks for the hint.

**Comment by Romeeey** [comment]
Did you document (eg in a ticket) what the exact mechanism with the v and c suffix exactly is? 
I would recommend to do so.

Implementation looks good.

**Pull request merged**